.ruff.toml

```diff
@@ -87,3 +87,16 @@ max-args = 8
 
 [lint.pydocstyle]
 convention = "numpy"
+
+[lint.isort]
+known-third-party = [
+    "matplotlib",
+    "munch",
+    "numpy",
+    "pandas",
+    "pygal",
+    "scipy",
+    "shapely",
+    "pytz",
+    "typing_extensions",
+]
```

Comment:
I'm not sure, but is there a way this can reference requirements directly?

Reply:
not that I know of